### PR TITLE
Support non-standard property-setting method using Sets annotation

### PR DESCRIPTION
**Pull request by davidchau**
Support non-standard property-setting methods (without converter)

### CODE REVIEW

File: src/test/java/com/fluentinterface/BuilderProxyTest.java [hunk 1]
```diff
@@ -187,7 +187,7 @@ public void shouldSetPropertyValueArrayOfPrimitivesToArrayOfPrimitives() {
                 .withAgesOfMarriages(23, 45)
                 .build();
 
-        assertThat(built.getAgesOfMarriages(), is(new int[] {23, 45}));
+        assertThat(built.getAgesOfMarriages(), is(new int[]{23, 45}));
```
Comment (davidmarquis) [inline src/test/java/com/fluentinterface/BuilderProxyTest.java]: auto-reformat?

File: README.md [hunk 2]
```diff
@@ -209,7 +223,7 @@ Here are some features I'd like to eventually add to the project:
     Allows for use cases where inputs as strings are used directly (I am mostly thinking of functional testing frameworks
     like Fitnesse (fixtures) or Cucumber)
  * Support for specifying custom handlers for builder methods. Could allow: `aPerson().named("Joe").aged(45).build()`
- * Configuration of the non-standard builder methods through annotations. Something like:
+ * Support conversion in `Sets` annotation. Something like:
 
 ```java
 public interface PersonBuilder extends Builder<Person> {
```
Comment (davidmarquis) [inline README.md]: I think this code example is not needed anymore.

File: README.md [hunk 3]
```diff
@@ -209,7 +223,7 @@ Here are some features I'd like to eventually add to the project:
     Allows for use cases where inputs as strings are used directly (I am mostly thinking of functional testing frameworks
     like Fitnesse (fixtures) or Cucumber)
  * Support for specifying custom handlers for builder methods. Could allow: `aPerson().named("Joe").aged(45).build()`
- * Configuration of the non-standard builder methods through annotations. Something like:
+ * Support conversion in `Sets` annotation. Something like:
```
Comment (davidmarquis) [inline README.md]: Basic conversion is available in the library when setting the target object's property value, I don't see a difference with a property that is being set from a hint via the annotation. Or I overlooked something?

Comment (davidchau) [inline README.md]: I was not sure if that would still be a wanted feature. I only reword it as I only implemented the `@Sets` annotation.
I can remove this if not needed anymore

File: src/test/java/com/fluentinterface/BuilderProxyTest.java (hunk 1 above)
Comment (davidchau) [inline src/test/java/com/fluentinterface/BuilderProxyTest.java]: Yes

File: README.md (hunk 3 above)
Comment (davidmarquis) [inline README.md]: yep, better to remove now I think!

**Comment by davidchau** [comment]
done

**Comment by davidmarquis** [comment]
Released version 1.3.1 that includes this change, readily available here:
https://oss.sonatype.org/content/repositories/releases/com/github/davidmarquis/fluent-interface-proxy/

Will get synced to Maven central in the next hours or so

**Comment by davidchau** [comment]
cool thanks

**Pull request merged**